src/plugins/layout/stateprovider.ts

```diff
@@ -138,7 +138,7 @@ function getAttributes(element) {
 
     // If we are masking text, also mask it from input boxes as well as alt description
     if (!config.showText && attributeMaskList.indexOf(attrName) >= 0) {
-      stateAttributes[attr.name] = attr.value.replace(/\S/gi, "*");
+      stateAttributes[attr.name] = attr.value.replace(/./g, "*");
```

Comment:
Small improvement: Refactor the masking into common code so we can update/change more easily.